File: codechef10.c
#include <stdio.h>
int cmpfunc (const void * a, const void * b)
{
   return ( *(int*)a - *(int*)b );
}

int main()
{
	/* code */
	int testCase,i=0,n=0,count = 0,t=0,j;
	scanf("%d",&testCase);
	int rackArray[30],pieArray[30],countArray[30];
	while(testCase>t)
	{
		scanf("%d",&n);
		for ( i = 0; i < n; ++i)
		{
			/* code */
			scanf("%d",&pieArray[i]);
		}
		for ( i = 0; i < n; ++i)
		{
			/* code */
			scanf("%d",&rackArray[i]);
		}
		qsort(pieArray, n, sizeof(int), cmpfunc);
		qsort(rackArray, n, sizeof(int), cmpfunc);
		j=0;
		for ( i = 0; i < n; ++i)
		{
			/* code */
			if(pieArray[j]<=rackArray[i]){
			count = count + 1;
			j=j+1;
		}
		}
		countArray[t] = count;
		count = 0;
		t=t+1;
	}
	for ( i = 0; i < t; ++i)
	{
		/* code */
		printf("%d\n",countArray[i] );
	}
	return 0;
}

File: codechef10.py
testCase = int(raw_input())
pieArray = []
rackArray = []
i=0
n=0
count = 0
t=0
countArray = []
while testCase>t:
	n = int(raw_input())
	for i in xrange(0,n):
		temp = int(raw_input())
		pieArray.insert(i,temp)
	for i in xrange(0,n):
		temp = int(raw_input())
		rackArray.insert(i,temp)
	rackArray.sort()
	pieArray.sort()
	
	
	j=0
	for i in xrange(0,n):
		if(pieArray[j]<=rackArray[i]):
			count = count + 1
			j=j+1

	
	
	countArray.insert(t,count)
	count=0
	t=t+1
for x in xrange(0,t):
	print countArray[x]